views/components/form-fields-next/macros.njk

```diff
@@ -656,3 +656,22 @@
             {% endif %}
     </div>
 {% endmacro %}
+
+{% macro continueButton() %}
```

Comment:
Probably prefer to make this a standalone macro, rather than in `form-fields` as it's not related to the fields. Either dedicated macro, or maybe inline it into the step?

Reply:
Done!